.github/workflows/ci-unwelcome-words.yml

```diff
@@ -1,20 +1,25 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  pull_request:
 
 jobs:
   test:
+    if: github.actor != 'dependabot[bot]'
```

Comment:
Is it possible that dependabot actor name will be changed in the future? Can we rely on some constant?

Reply:
I don't find any constant https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions